src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/ComparisonHelper.java

```diff
@@ -5,38 +5,80 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.Objects;
+
+import uk.gov.companieshouse.api.model.common.Address;
+import uk.gov.companieshouse.api.model.officers.FormerNamesApi;
+import uk.gov.companieshouse.api.model.officers.OfficerRoleApi;
 import uk.gov.companieshouse.api.model.utils.AddressApi;
 import uk.gov.companieshouse.overseasentitiesapi.model.dto.AddressDto;
 import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.PersonName;
 
+import static uk.gov.companieshouse.overseasentitiesapi.utils.FormerNameConcatenation.concatenateFormerNames;
+
 public class ComparisonHelper {
 
   private ComparisonHelper() {
     throw new IllegalAccessError("Use the static method designation");
```

Comment:
No need for this throw. Just lowers test coverage (unless you added a test for it).

Reply:
I know but test coverage low.